spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+describe GradeEntryFormsController do
+
+  before :each do
+    # Authenticate user is not timed out, and has administrator rights.
+    allow(controller).to receive(:session_expired?).and_return(false)
+    allow(controller).to receive(:logged_in?).and_return(true)
+    allow(controller).to receive(:current_user).and_return(build(:admin))
+
+    #allow(GradeEntryForm).to receive(:find).and_return()
+
+  end
+
+  let(:grade_entry_form) { create(:grade_entry_form) }
+
+
+
+  context 'CSV_Uploads' do
+
+    before :each do
+      @file_without_extension = fixture_file_upload('spec/fixtures/files/grade_entry_upload_empty_file')
+      @file_wrong_format = fixture_file_upload('spec/fixtures/files/grade_entry_upload_wrong_format.xls')
+      @file_bad_csv = fixture_file_upload('spec/fixtures/files/grade_entry_upload_bad_csv.csv')
+      @file_good_csv = fixture_file_upload('spec/fixtures/files/grade_entry_upload_good_test.csv')
+    end
+
+   it 'does not accept an empty file' do
+      post :csv_upload, { id: grade_entry_form }, { :upload => @file_without_extension }
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.